lib/helpers/finalizeSpecificationObject.js

```diff
@@ -26,6 +27,32 @@ function cleanUselessProperties(inputSpec) {
   return improvedSpec;
 }
 
+/**
+ * AsyncAPI specification validator does not accept empty values for a few properties.
+ * Solves validator error: "Schema error should NOT have additional properties"
+ * @function
+ * @param {object} inputSpec - The asyncapi specification
+ * @param {object} improvedSpec - The cleaned version of the inputSpec
+ */
+function cleanUselessPropertiesAsyncApi(inputSpec) {
```

Comment:
As noted in the comments, do not eagerly share parser and then move things back and forth